src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -24,7 +24,8 @@ public abstract class LbfgsTrainerBase<TOptions, TTransformer, TModel> : Trainer
       where TOptions : LbfgsTrainerBase<TOptions, TTransformer, TModel>.OptionsBase, new()
     {
         /// <summary>
-        /// Base options for <a href='https://en.wikipedia.org/wiki/Limited-memory_BFGS'>L-BFGS</a>-based trainers.
+        /// Options for the <see cref="LbfgsTrainerBase{TOptions, TTransformer, TModel}"/>.
```

Comment:
>Options [](start = 12, length = 7)

Suggestion:
`Base Options class for trainer estimators deriving from ...`.
(For the SDCA base class too).